rust/tool/src/install.rs

```diff
@@ -307,7 +315,16 @@ impl Installer {
         ];
 
         for (from, to) in paths {
-            if newer_systemd_boot(from, to)? || !&self.key_pair.verify(to) {
+            let newer_systemd_boot_available = newer_systemd_boot(from, to)?;
+            if newer_systemd_boot_available {
+                log::info!("Updating systemd-boot...")
+            };
+            let systemd_boot_is_signed = &self.key_pair.verify(to);
+            if !systemd_boot_is_signed {
+                log::warn!("systemd-boot is not signed. Replacing it with a signed binary...")
+            };
```

Comment:
I don't love this. Do you have any idea how to make this more concise?

Reply:
I think it's good enough. I personally would move it into the `if` statement that does the actual installation, but it doesn't really matter.